pyproject.toml

```diff
@@ -54,9 +54,9 @@ dependencies = [
 
 [project.optional-dependencies]
 tf = [
-    "tensorflow>=2.9.0,<3.0.0",  # cf. https://github.com/mindee/doctr/issues/454
+    "tensorflow>=2.11.0,<3.0.0",  # cf. https://github.com/mindee/doctr/issues/454
     "tensorflow-addons>=0.17.1",
-    "tf2onnx>=1.9.2",
+    "tf2onnx>=1.14.0",
```

Comment:
Same here (if there's any) and perhaps add an upper bound? (<2.0.0 I would suggest)